CMakeLists.txt

```diff
@@ -440,7 +440,7 @@ if(FEAT_TUI)
 endif()
 
 find_package(LibVterm REQUIRED)
-include_directories(SYSTEM ${LIBVTERM_INCLUDE_DIRS})
+include_directories(SYSTEM ${LibVterm_INCLUDE_DIRS})
```

Comment:
why change the case? better to be consistent with most of our existing cmake code.

Reply:
It is a side-effect of using the library.
Yes.
We could still set it manually, or have an option/flag for it to be done in LibFindMacros - but after all it would have avoided the recent confusion with REQUIRED not being handled due to case etc.

While it adds a bit more noise (and might affect distributions/users), it is good to have it more consistent.

I plan to use LibFindMacros for the other Find modules, so this will have an effect there also.

Reply:
It appears we can (and should) keep the uppercase variables (like done in https://github.com/neovim/neovim/pull/10681).